src/openlifu/db/volume.py

```diff
@@ -0,0 +1,73 @@
+import json
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Optional
+
+from openlifu.util.json import PYFUSEncoder
+
+
+@dataclass
+class Volume:
+    """
+    Class representing a volume
+
+    ivar id: ID of the volume
+    ivar name: Name of the volume
+    """
+    id: Optional[str] = None
+    name: Optional[str] = None
```

Comment:
id and name are good but there needs to be another attribute: `filename`. This is needed by Database in order to later recover the volume path